site/en/blog/show-picker/index.md

```diff
@@ -0,0 +1,203 @@
+</figure>
+
+{% Aside %}
+Unlike a third-party picker widget, a browser picker is familiar to the user,
+has great specific platform support, and is always maintained as part of the
+browser.
+{% endAside %}
+
+## How to show a picker {: #how-to }
+
+Calling `showPicker()` on an `<input>` element returns a promise that resolves
+when a browser picker is shown to the user. It must be called in response to a
+user gesture such as a touch gesture or mouse click; otherwise it will reject.
+For security reasons,it will also reject when it's called in a cross-origin
+iframe.
+
+A browser picker is shown when the `<input>` element is one of these types:
+`"date"`, `"month"`, `"week"`, `"time"`, `"datetime-local"`, `"color"`, and
+`"file"`.
+
+The example below shows you how to open a browser date picker.
+
+```html/9
+<input type="date">
+<button>Show the date picker</button>
+
+<script>
+  const button = document.querySelector("button");
+  const dateInput = document.querySelector("input");
+
+  button.addEventListener("click", async () => {
+    try {
+      await dateInput.showPicker();
+      // A date picker is shown.
+    } catch (error) {
+      // Use external library when this fails.
+    }
+  });
+</script>
+```
+
+A browser picker may also be shown if the `<input>` element has some suggestions
+powered by `<datalist>` or `"autocomplete"`.
+
+The example below shows you how to open a browser picker with `<datalist>`.
+
+```html/16
+<datalist id="ice-cream-flavors">
+  <option value="Chocolate"> </option>
+  <option value="Coconut"> </option>
+  <option value="Mint"> </option>
+  <option value="Strawberry"> </option>
+  <option value="Vanilla"> </option>
+</datalist>
+<input type="text" list="ice-cream-flavors">
+<button>Show the suggestions</button>
+
+<script>
+  const button = document.querySelector("button");
+  const iceCreamFlavorsInput = document.querySelector("input");
+
+  button.addEventListener("click", async () => {
+    try {
+      await iceCreamFlavorsInput.showPicker();
+      // A picker containing some ice cream flavors is shown.
+    } catch (error) {
+      // Use external library when this fails.
+    }
+  });
+</script>
+```
+
+## Demo
+
+A playground demo is available at
+[https://show-picker.glitch.me/demo.html][demo] for you to play with all
+supported pickers by the browser.
+
+{% Video src="video/vvhSqZboQoZZN9wBvoXq72wzGAf1/MMPmrnJnMvFX3eMCUWK9.mov",
+autoplay="true", muted="true", loop="true", class="screenshot" %}
+
+## Browser support
+
+At the time of writing, `showPicker()` is available in Chrome&nbsp;99.
+
+## What's next {: #future }
+
+At the time of writing, `showPicker()` is new to the web platform. Some
+additional work may be needed in the future:
+
+- We may want to add a similar `showPicker()` to the `<select>` element in the
+  future, if web developers ask for it.
+- It's possible `closePicker()` might be useful, and we could consider adding
+  that if web developers ask for it again.
```

Comment:
```suggestion
  that if web developers ask for it.
```

Reply:
I think you intended 'again' to refer to the previous bullet. Anglophones would hear this as, "developers have already asked for this, but were turned down."

Reply:
Thanks! Done